x-pack/plugins/enterprise_search/README.md

```diff
@@ -119,7 +119,13 @@ There are 3 ways you can spin up the required environments to run our Cypress te
    - Enterprise Search:
      - Nothing extra is required to run Cypress tests, only what is already needed to run Kibana/Enterprise Search locally.
 2. Running Cypress against Kibana's functional test server:
-   - TODO
+   - :information_source: While we won't use the runner, we can still make use of Kibana's functional test server to help us spin up Elasticsearch and Kibana instances.
```

Comment:
Just want to highlight that I'm not 100% certain if this will stay in, but we can remove this section/commit later if we decide against using the functional test server in CI (vs., e.g. docker images)